Docs/operator/LocalOperator.md

```diff
@@ -95,9 +95,9 @@ that the matrix element O(v,v') is different from zero. In general there
 will be several different connected visible units satisfying this 
 condition, and they are denoted here v'(k), for k=0,1...N_connected.
 
-|Argument|Type|                   Description                    |
-|--------|----|--------------------------------------------------|
-|v       |    |A constant reference to the visible configuration.|
+|Argument|         Type          |                   Description                    |
+|--------|-----------------------|--------------------------------------------------|
+|v       |numpy.ndarray[float64[m|A constant reference to the visible configuration.|
```

Comment:
maybe some residual glitch with types here?

Reply:
I think it's cutting off the type after a certain number of characters. I will look into this

Reply:
I think you're right, there are other types that are longer.

Reply:
@gcarleo I fixed this in the commit 1f61edf. During the parsing of the function signature is was splitting the string of the entire signature using `,` as a delimiter. Since some of the the objects (e.g. `numpy.ndarray[float64[m, 1]`) have commas in their type, the type names for some arguments were getting garbled.

@ooreilly I changed the `parse_signature()` function in `parse.py`. Let me know what you think. I've tried to make it a bit more general, so it can handle cases with an unknown number of `:`. I think this will be important for a several of our uses (e.g. the type of a sampler `netket::AbstractSampler<netket::AbstractMachine<std::complex<double> > >`)